config/src/config/consensus_config.rs

```diff
@@ -132,31 +132,31 @@ impl Default for ConsensusConfig {
             vote_back_pressure_limit: 30,
             pipeline_backpressure: vec![
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 1000,
+                    back_pressure_pipeline_latency_limit_ms: 2000,
                     max_sending_block_txns_override: 10000,
                     max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backpressure_proposal_delay_ms: 100,
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 1500,
+                    back_pressure_pipeline_latency_limit_ms: 2500,
                     max_sending_block_txns_override: 10000,
                     max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backpressure_proposal_delay_ms: 200,
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 2000,
+                    back_pressure_pipeline_latency_limit_ms: 3000,
                     max_sending_block_txns_override: 10000,
                     max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backpressure_proposal_delay_ms: 300,
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 2500,
+                    back_pressure_pipeline_latency_limit_ms: 3500,
```

Comment:
We expect 300ms * 2 increase in latency compared to the previous (@ 3000 ms) so it seems like we expect to cross over to this value unless there is a sudden drop in the latency. Is this what we want? Or should this threshold be something > 3000 + 300*2 ?

Reply:
this is not additional 300ms, this is if we cross 3s with delay included. 

so crossing 3000 adds additional 200ms. so there is additional 300ms of gap before the next level - things are not escalating automatically (it is just that gaps are between levels are 300ms not 500ms as the values would suggest.

Reply:
Ah, I see the increase will be the delta * 2.